cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -982,6 +991,11 @@
     public static final String HIVE_AUTHORIZATION_SQL_STD_AUTH_CONFIG_WHITELIST_APPEND =
       "hive.security.authorization.sqlstd.confwhitelist.append";
 
+    /** Whether to use latest hive version when hive version is unknown */
+    public static final String HIVE_VERSION_FOR_UNKNOWN_VERSION = "hive.version";
```

Comment:
Why is `hive.version` also an underlying implementation? What can be an alternative name?